src/cli/mod.rs

```diff
@@ -27,7 +27,7 @@ impl MultiCall {
         };
         match exe_name.as_bytes() {
             b"bootupctl" => MultiCall::Ctl(bootupctl::CtlCommand::from_iter(args)),
-            b"bootupd" | _ => MultiCall::D(bootupd::DCommand::from_iter(args)),
+            _ => MultiCall::D(bootupd::DCommand::from_iter(args)),
```

Comment:
Hmm, I think this is `wildcard_in_or_patterns`.
Would having the three cases explicitly on different arms (`bootupctl` - `bootupd` - `_`) make `clippy` happier?